demo-app/src/main/java/cash/z/ecc/android/sdk/demoapp/demos/getbalance/GetBalanceFragment.kt

```diff
@@ -131,23 +134,28 @@ class GetBalanceFragment : BaseDemoFragment<FragmentGetBalanceBinding>() {
                     sharedViewModel.synchronizerFlow
                         .filterNotNull()
                         .flatMapLatest {
-                            it.orchardBalances.combine(it.exchangeRateUsd) { b, r ->
-                                b?.let {
-                                    b to
-                                        r.currencyConversion?.priceOfZec?.toBigDecimal()
+                            val account = it.getAccounts()[CURRENT_ZIP_32_ACCOUNT_INDEX]
```

Comment:
Ditto (I presume I can just search for this constant and find all such problematic instances later).